python/test/unit/fem/test_petsc_nonlinear_assembler.py

```diff
@@ -403,15 +385,10 @@ def blocked_solve():
             p.interpolate(initial_guess_p)
 
             x = create_vector_block(F)
-            scatter_local_vectors(
-                x,
-                [u.x.petsc_vec.array_r, p.x.petsc_vec.array_r],
-                [
-                    (u.function_space.dofmap.index_map, u.function_space.dofmap.index_map_bs),
-                    (p.function_space.dofmap.index_map, p.function_space.dofmap.index_map_bs),
-                ],
-            )
-            x.ghostUpdate(addv=PETSc.InsertMode.INSERT, mode=PETSc.ScatterMode.FORWARD)
+
+            # Assign u, p values to x
```

Comment:
I think this comment can go, the code below is self-explanatory.

Reply:
Or is it "assign x values to u, p" :)